_posts/2020-06-20-ArrowToTheKneeEditingUnorderedMultisetItems.md

```diff
@@ -7,78 +7,78 @@ tags: [Arrow to the Knee, C++, Containers]
 ---
 
 Have you ever thought of editing `std::unordered_multiset` items while iterating over it? Does it even make sense?
-In this post I will tell you why and how I tried to do it and what happened to my code afterwards.
+I will tell you why and how I tried doing it and what happened to my code afterwards in this post.
```

Comment:
From what I know, `try to do` means `challenge` and I meant that it was a challenge here.